syft/pkg/nix_store_metadata.go

```diff
@@ -0,0 +1,24 @@
+package pkg
+
+import (
+	"sort"
+
+	"github.com/scylladb/go-set/strset"
+)
+
+type NixStoreMetadata struct {
+	// Hash is the prefix of the nix store basename path
+	Hash string `mapstructure:"hash" json:"hash"`
+
+	// Output allows for optionally specifying the specific nix package output this package represents (for packages that support multiple outputs)
```

Comment:
The default output also doesn't have this. Should probably be added in the comment.